lib/json_logic/operation.rb

```diff
@@ -144,11 +144,5 @@ def self.is_standard?(operator)
     def self.is_iterable?(operator)
       ['filter', 'some', 'all', 'none', 'in', 'map', 'reduce'].include?(operator.to_s)
     end
-
-    def self.add_operation(operator, function)
```

Comment:
Didn't seem like this was doing anything, so I took it out.
Happy to add it back if desired.